samples/integration/vcpkg-all-smoke/src/main.cpp

```diff
@@ -25,51 +26,57 @@ using namespace Azure::Storage::Blobs;
 using namespace Azure::Storage::Queues;
 using namespace Azure::Storage::Files::DataLake;
 using namespace Azure::Storage::Files::Shares;
+using namespace Azure::Security::Attestation;
 
 int main()
```

Comment:
I won't complain about program, I think it brings value.
However, I do think the value it brings is super minimal and basically reduced to `check we can compile clients into one binary`
Remember the linker will take only what's used by the application, so we are only taking the client definitions (hpp files and the constructor implementation).

Hopefully we could call operations from the clients to see them all calling I/O one after eachother and also at the same time.  That would be fun!

Reply:
initially i was doing that , but after some discussion with some people involved in this, the decission was made to not call any APIs.